test/unit/attributes.js

```diff
@@ -1117,7 +1139,7 @@ QUnit.test( "val(select) after form.reset() (Bug #2551)", function( assert ) {
 } );
 
 QUnit.test( "select.val(space characters) (gh-2978)", function( assert ) {
-	assert.expect( 35 );
+	assert.expect( 37 );
```

Comment:
Oh, so these editions are unrelated to bug fix? I would move them to different commit, but not feeling strong about this